views/pages/fixture_editor.js

```diff
@@ -32,12 +38,12 @@ module.exports = function(options) {
 
   str += '<section class="new-manufacturer-website">';
   str += '<label>Website</label>'
-  str += '<input />'
+  str += textInput(manProperties.website);
   str += '</section>'
 
   str += '<section class="new-manufacturer-comment">';
   str += '<label>Comment</label>'
-  str += '<input />'
+  str += textInput(manProperties.comment);
   str += '</section> or ';
   str += '<a href="#" class="use-existing-manufacturer">Use existing manufacturer</a>';
   str += '</section>'
```

Comment:
add a comment that describes which `<section>` is closed here (and again two lines below)